app/modules/encounters/resources.py

```diff
@@ -0,0 +1,115 @@
+# encoding: utf-8
+# pylint: disable=bad-continuation
+"""
+RESTful API Encounters resources
+--------------------------
+"""
+
+import logging
+
+from flask_login import current_user
+from flask_restplus_patched import Resource
+from flask_restplus._http import HTTPStatus
+
+from app.extensions import db
+from app.extensions.api import Namespace
+from app.extensions.api.parameters import PaginationParameters
+from app.modules.users import permissions
+
+
+from . import parameters, schemas
+from .models import Encounter
+
+
+log = logging.getLogger(__name__)  # pylint: disable=invalid-name
+api = Namespace('encounters', description="Encounters")  # pylint: disable=invalid-name
+
+
+@api.route('/')
+@api.login_required(oauth_scopes=['encounters:read'])
+class Encounters(Resource):
```

Comment:
While I am certainly ok with the existence of encounter endpoints, it might be helpful to note now that I plan to request occurrences/sightings most of the time instead.

Reply:
That is ok, we will be adding endpoints for all of the above in Houston (albeit not all will have data models in Houston's database)